packs/data/extinction-curse-bestiary.db/pin-tingwheely.json

```diff
@@ -271,10 +271,9 @@
                     "value": true
                 },
                 "level": {
-                    "value": 2
+                    "value": 3
                 },
                 "location": {
-                    "heightenedLevel": 3,
                     "value": "aGUovHznkIyYG8m6"
```

Comment:
What happened here? Why did they heighten information vanish?

Reply:
Let me look into this again.

Reply:
Fixed with [5619553](https://github.com/foundryvtt/pf2e/pull/2726/commits/561955346d8ad8680ffb26469141e37bb70dcc81)